service-deployment/bootstrap/prometheus/node-exporter-ds.yaml.template

```diff
@@ -65,37 +65,6 @@ spec:
         - mountPath: {{ clusterinfo[ 'dataPath' ] }}/prometheus
           name: collector-mount
         name: gpu-exporter
-      - image: google/cadvisor:v0.29.0 
-        securityContext:
-          privileged: true
-        volumeMounts:
-        - mountPath: /dev
-          name: device-mount
-        - mountPath: /var/drivers
-          name: driver-path
-        - name: rootfs
-          mountPath: /rootfs
-          readOnly: true
-        - name: var-run
-          mountPath: /var/run
-          readOnly: false
-        - name: sys
-          mountPath: /sys
-          readOnly: true
-        - name: docker
-          mountPath: /var/lib/docker
-          readOnly: true
-        env:
-        - name: LD_LIBRARY_PATH
-          value: "/var/drivers/nvidia/current/lib:/var/drivers/nvidia/current/lib64"
-        args:
-          - '--port={{ clusterinfo['prometheusinfo']['cadvisor_port'] }}' 
```

Comment:
>cadvisor_port [](start = 54, length = 13)

please remove this in clusterconfig.yaml accordingly.

Reply:
done